src/tests/ftest/launch.py

```diff
@@ -1546,14 +1546,22 @@ def resolve_debuginfo_dnf(pkg):
     dnf_base = dnf.Base()
     dnf_base.conf.assumeyes = True
     dnf_base.read_all_repos()
-    dnf_base.fill_sack()
+    try:
+        dnf_base.fill_sack()
+    except OSError as error:
+        print("Got an OSError trying to fill_sack(): ", error)
+        raise RuntimeError("resolve_debuginfo_dnf() failed: ", error)
```

Comment:
(pylint-raise-missing-from) Consider explicitly re-raising using the 'from' keyword